proposals/lip-0071.md

```diff
@@ -280,9 +291,9 @@ The Dynamic Block Rewards module does not execute any logic during the genesis s
 Before the transactions of a block `b` are executed, the following logic is executed:
 
 ```python
-def beforeTransactionsExecute(b: Block) -> None:
+def beforeTransactionsExecute(b: Block, blockAssets: BlockAssets) -> None:
```

Comment:
The hook `beforeTransactionsExecute` can have only one argument, `b: Block`. You should obtain the assets as `b.assets`, keep in mind the format that you end up choosing for the assets (decoded / not decoded).

Reply:
Agree with Sergey. So just use `b.blockAssets` as argument for `getDynamicBlockReward`

Reply:
Updated.